vcbuild.bat

```diff
@@ -55,13 +55,14 @@ set http2_debug=
 set nghttp2_debug=
 set link_module=
 set no_cctest=
+set cctest=
 set openssl_no_asm=
 set doc=
 
 :next-arg
 if "%1"=="" goto args-done
 if /i "%1"=="debug"         set config=Debug&goto arg-ok
-if /i "%1"=="release"       set config=Release&set ltcg=1&set "pch="&goto arg-ok
+if /i "%1"=="release"       set config=Release&set ltcg=1&set "pch="&set cctest=1&goto arg-ok
```

Comment:
maybe we should create a `ci` target, since the "real" release doesn't need `cctest` either.

Reply:
SGTM, this should converge to be like the Makefile, including the distinction between compiling and js-only targets. Currently `cctest` runs and the addons are compiled everywhere, unnecessarily.